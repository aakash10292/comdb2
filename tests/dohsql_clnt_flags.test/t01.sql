set timezone UTC
insert into t1 (id, date) values (1, cast("2007-10-01" as datetime))
insert into t1 (id, date) values (2, cast("2007-10-02" as datetime))
insert into t1 (id, date) values (3, cast("2007-10-02" as datetime))
insert into t1 (id, date) values (4, cast("2007-10-02" as datetime))
select count(*) from t1 where date='2007-10-02' union all select count(*) from t1 where date='2007-10-02'
set timezone US/Eastern
select count(*) from t1 where date='2007-10-02' union all select count(*) from t1 where date='2007-10-02'
