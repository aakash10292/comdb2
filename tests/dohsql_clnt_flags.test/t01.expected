(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(rows inserted=1)
(count(*)=3)
(count(*)=3)
(count(*)=0)
(count(*)=0)
